requirements/ingest/chroma.in

```diff
@@ -1,6 +1,10 @@
 -c ../deps/constraints.txt
 -c ../base.txt
 chromadb
-importlib-metadata>=7.1.0
+importlib-metadata>=8.2.0
```

Comment:
`opentelemetry-api` has `importlib-metadata` pinned [here](https://github.com/open-telemetry/opentelemetry-python/blob/main/opentelemetry-api/pyproject.toml) to `"importlib-metadata >= 6.0, <= 8.2.0",`

Reply:
Other extras were resolving to 8.2.0, but chroma was resolving to 8.0.0 for some reason. Pinned 8.2.0 as a minimum version to force the chroma dependency to match the others.